crowbar_framework/app/controllers/api/ceph_controller.rb

```diff
@@ -14,23 +14,22 @@
 # limitations under the License.
 #
 
-class Api::StoragesController < ApiController
+class Api::CephController < ApiController
   before_action :set_ceph
 
-  api :GET, "/api/storages", "List all Ceph storages"
+  api :GET, "/api/ceph", "Show the Ceph object"
```

Comment:
right now nothing, but it should display some general data (see also GET /api/crowbar). what about returning the ceph version for a start?